modules/QnA/resources/qnaService.js

```diff
@@ -671,18 +671,34 @@ DAL for Q&A Module
                             disableModule = false;
                             announcementOnly = false;
                         }
-
-                        if (cuePoint.tags && cuePoint.tags.indexOf("WEBCASTSTATETAG")>=0) {
-                            try {
-                                var webCastingState=JSON.parse(cuePoint.code);
-                                disableModule=!webCastingState["QnA"];
-                                announcementOnly=!disableModule && webCastingState["AnnouncementsOnly"];
+                        //for BC supporting both new and old QnA settings cue points
+                        if(cuePoint.tags) {
+                            //old QnA settings cue point convention //todo [sa] remove after releasing new producer version
+                            if (cuePoint.tags.indexOf("WEBCASTSTATETAG") >= 0) {
```

Comment:
@esakal will be removed only after both player and webcasting app will get updated